### PR TITLE
Add packages

### PR DESCRIPTION
**Pull request by TBomberman**
1. Added v1-core and v1-periphery as yarn packages.
2. Removed old abis
3. Updated old abi references to the new packages.
4. Withdraw will be broken as it is using an old hackathon abis. To be connected again with this task: https://www.notion.so/S4-10-03-10-14-39a87ba77fc4445287762e021b523b30?p=3df17d65f1654441801699ed253ad154&pm=s
5.ty @dam417  or @Pochetes

### CODE REVIEW

File: src/hooks/useWithdrawLiquidityHandler.tsx [hunk 1]
```diff
@@ -52,11 +50,11 @@ export const useWithdrawLiquidityHandler = () => {
...
-          setDepPool(new ethers.Contract(address, DepPool.abi, provider.getSigner(accountInfo.address)))
-        } else {
-          setDepPool(new ethers.Contract(address, DepPool.abi, provider))
-        }
+        // if (accountInfo?.address) {
+        //   setDepPool(new ethers.Contract(address, DepPool.abi, provider.getSigner(accountInfo.address)))
+        // } else {
+        //   setDepPool(new ethers.Contract(address, DepPool.abi, provider))
+        // }
       }
```
Comment (Pochetes) [inline src/hooks/useWithdrawLiquidityHandler.tsx:58]: why is this commented out?

File: src/hooks/useWithdrawLiquidityHandler.tsx [hunk 2]
```diff
@@ -113,11 +111,11 @@ export const useWithdrawLiquidityHandler = () => {
...
-        if (accountInfo && accountInfo?.address) {
-          setDepPool(new ethers.Contract(address, DepPool.abi, provider.getSigner(accountInfo?.address)))
-        } else {
-          setDepPool(new ethers.Contract(address, DepPool.abi, provider))
-        }
+        // if (accountInfo && accountInfo?.address) {
+        //   setDepPool(new ethers.Contract(address, DepPool.abi, provider.getSigner(accountInfo?.address)))
+        // } else {
+        //   setDepPool(new ethers.Contract(address, DepPool.abi, provider))
+        // }
```
Comment (Pochetes) [inline src/hooks/useWithdrawLiquidityHandler.tsx:118]: same with this

**Review by Pochetes — APPROVED** [review]
LGTM, made minor comments

File: src/hooks/useWithdrawLiquidityHandler.tsx (hunk 1 above)
Comment (TBomberman) [reply src/hooks/useWithdrawLiquidityHandler.tsx:58]: DepPool was a legacy object that connected to the hackathon abis. There was no close equivalent in the v1-periphery or v1-core. So I didn't know what to do with it. I commented it out to be compliant with the new packages and removing of v0-hackathan abis. This function's backend was never connected to the v1 contracts yet, so this commented code will be fixed when the connection is being worked on here To be connected again with this task: https://www.notion.so/S4-10-03-10-14-39a87ba77fc4445287762e021b523b30?p=3df17d65f1654441801699ed253ad154&pm=s.

File: src/hooks/useWithdrawLiquidityHandler.tsx (hunk 2 above)
Comment (TBomberman) [reply src/hooks/useWithdrawLiquidityHandler.tsx:118]: DepPool was a legacy object that connected to the hackathon abis. There was no close equivalent in the v1-periphery or v1-core. So I didn't know what to do with it. I commented it out to be compliant with the new packages and removing of v0-hackathan abis. This function's backend was never connected to the v1 contracts yet, so this commented code will be fixed when the connection is being worked on here To be connected again with this task: https://www.notion.so/S4-10-03-10-14-39a87ba77fc4445287762e021b523b30?p=3df17d65f1654441801699ed253ad154&pm=s.

File: src/hooks/useWithdrawLiquidityHandler.tsx (hunk 1 above)
Comment (Pochetes) [reply src/hooks/useWithdrawLiquidityHandler.tsx:58]: I see, make sure to add a `TODO` comment so the ones that work on it know where to make the changes

Comment (TBomberman) [reply src/hooks/useWithdrawLiquidityHandler.tsx:58]: ah right, will do it next time, ty for the reminder.

**Pull request merged**